packages/application-shell/src/components/handle-apollo-errors/handle-apollo-errors.js

```diff
@@ -0,0 +1,50 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { wrapDisplayName } from 'recompose';
+import { ReactReduxContext } from 'react-redux';
+import { handleActionError } from '@commercetools-frontend/actions-global';
+
+function handleApolloErrors(queryResultNames) {
+  return Component => {
+    class WrappedComponent extends React.Component {
+      static displayName = wrapDisplayName(Component, 'handleApolloErrors');
+      static propTypes = {
```

Comment:
I don't think we need to have an object and then spread? Can't we just `static propTypes = queryNames.reduce...` or was this for general extensibility of the prop-types?

Reply:
Yeah for making it less confusing if you need to extend it. I'm not sure if it would be the case here, so I guess we can remove the spread.